aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -261,7 +261,7 @@ impl AptosDebugger {
             &state_view_storage,
         )
         .unwrap();
-        let mut session = move_vm.new_session(&state_view_storage, SessionId::Void);
+        let mut session = move_vm.new_session(&state_view_storage, SessionId::Void, None);
```

Comment:
> passing None here means if you run session on top of something that wants to use your new functions, it will break?

Yes, calling the new transaction context native functions will abort if they are called when the user transaction context is `None` (e.g, it is not available for system transactions).

Reply:
I could be wrong, but it seems like `run_session_at_version` is an unused API. 

`execute_transactions_at_version` is the real entry point for the debugger and that uses AptosVM correctly.

Reply:
@vgao1996 , you're right. `run_session_at_version` doesn't seem to have any caller.

Reply:
Yeah in this case I'd argue that we can simply remove `run_session_at_version` altogether.

Reply:
I removed `run_session_at_version`.